src/components/SidebarAdmin/index.tsx

```diff
@@ -243,6 +244,39 @@ function SideBar() {
             </StyledListItemButton>
           )}
         </NavLink>
+
+        <NavLink to={urls.ADMIN_STRIPE}>
+          {({ isActive }) => (
+            <StyledListItemButton selected={isActive}>
+              <Box
+                display="flex"
+                justifyContent="space-between"
+                alignItems="center"
+                width="100%"
+              >
+                <ListItemIcon sx={{ minWidth: '24px' }}>
+                  <StripeIcon />
+                </ListItemIcon>
+                {isActive ? (
+                  <StyledChat>
+                    <Typography
+                      variant="subtitle1"
+                      sx={{ marginLeft: '12px', fontSize: '16px' }}
+                    >
```

Comment:
Use fontSize, fontWidth from theme, plz.

Reply:
Such styles are used in the tabs above, so I used the same in my tab